System.IO.Abstractions.TestingHelpers/MockFileSystem.cs

```diff
@@ -24,11 +25,13 @@ public MockFileSystem(IDictionary<string, MockFileData> files, string currentDir
                 currentDirectory = XFS.Path(DEFAULT_CURRENT_DIRECTORY);
             }
 
+            var defaultTempDirectory = XFS.Path(DEFAULT_TEMP_DIRECTORY);
```

Comment:
Just a minor nit: I’d leave out the „default“ prefix here as there is no way to change this value form the outside.